README.md

```diff
@@ -134,23 +134,70 @@ interface "wasi:kv/data/set-many" {
   // Notice that this interface is non-atomic, meaning that the key-value pairs
   // may not be consistent.
   use { Error, key } from "wasi:kv/types"
-  
+
   set-many: func(key_values: stream<(key, stream<u8>)>) -> result<_, Error>
 }
 
+interface "wasi:kv/data/range" {
+  // request using a range of keys
+  use { Error, payload, key-range, key-value } from "wasi:kv/types"
```

Comment:
Need to import `key`

Reply:
Considering that `use` is not currently supported within `interface`s, I've chosen to omit the statements for now~ up to you if you want to do the same, Steve.

Reply:
commented out for now, until the `use` syntax is resolved